USAGE.md

```diff
@@ -0,0 +1,104 @@
+# Amazon Alexa Smart Home Skill for openHAB 2
+
+Amazon certified openHAB2 as a [smarthome skill for Alexa](https://www.amazon.com/openHAB-Foundation/dp/B01MTY7Z5L).
+This page describes how to use the skill certified by Amazon.
+This skill allows you to connect your openHAB setup through the [myopenHAB.org](http://myopenHAB.org) cloud service to Amazon Alexa.
+(See [README.md](https://github.com/openhab/openhab-alexa/blob/master/README.md) for other setup options regarding your own openHAB cloud.)
+
+With this skill you can control items that respond to on/off and numeric commands.
```

Comment:
I would change this to read

"The openHAB Alexa skill can control items that respond to on/off and numeric commands and is supported for English (U.S.), English (U.K.), and German languages.

Reply:
Think the first passage is OK as it is.